core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.ts

```diff
@@ -268,6 +439,12 @@ popup(state){
             this.isLoading = false;
           }
           else {
+            this.safeName = response.data.environment[0].metadata.safe.name
```

Comment:
Please gracefully fail when accessing fields within nested objects. Generally, it is good practice to assume uncertainty. Please update all scenarios in new code where this may be the case.

Reply:
done

Reply:
done